components/PlayerCard/index.js

```diff
@@ -1,11 +1,14 @@
 import Image from "next/image";
 import styled from "styled-components";
+import Link from "next/link";
```

Comment:
For readability, I was told the next imports should be next to each other, so first the next/image, then the next/link, then the styled components